core/Version.php

```diff
@@ -20,7 +20,7 @@ final class Version
      * The current Matomo version.
      * @var string
      */
-    const VERSION = '4.4.0-rc1';
+    const VERSION = '4.4.0-rc2';
```

Comment:
Guess this and the update script should be changed to `rc3` as `rc2` was already released.